setup.py

```diff
@@ -34,6 +34,7 @@ def readme():
         "django-language-field",
         "chardet",
         "tablib",
+        "mailchimp_marketing",
```

Comment:
I think this would be an "extra_requires" entry. E.g. ```{ ...., "mailer_integration": "mailchimp_marketing"}```
And the in the requirements.txt of the project you would write ```basxconnect[mailer_integration]```